src/applications/vaos/actions/appointments.js

```diff
@@ -11,18 +11,11 @@ import { resetDataLayer } from '../utils/events';
 
 import {
   getCancelReasons,
-  getFacilitiesBySystemAndTypeOfCare,
   getRequestMessages,
   updateAppointment,
   updateRequest,
 } from '../api';
 
-import {
-  getOrganizations,
-  getRootOrganization,
-  getSiteIdFromOrganization,
-} from '../services/organization';
-
```

Comment:
My fault.   Should have removed these after moving express care into it's own actions file.